src/components/NavBar.spec.js

```diff
@@ -0,0 +1,10 @@
+import { render } from '@testing-library/react'
+import NavBar from './NavBar'
+
+describe('it calls onClickSearch when clicking on the navbar', () => {
+  it('renders', () => {
+    const onClickSearch = jest.fn()
+
+    render(<NavBar onClick={onClickSearch} />)
+  })
```

Comment:
I would specify the test more and test of onClickSearch is called when clicking on the Search button and onClickHome when clicking on the Home button